src/test/resources/FHIR_oBDS_Expected-File-4.xml

```diff
@@ -27,6 +27,160 @@
          <url value="Patient/1302b6d11c5b628d"/>
       </request>
    </entry>
+   <entry>
+      <fullUrl value="http://example.com/Specimen/bio644974f7c86b960a"/>
+      <resource>
+         <Specimen>
+            <id value="bio644974f7c86b960a"/>
+            <meta>
+               <profile value="https://fhir.bbmri.de/StructureDefinition/Specimen"/>
+            </meta>
+            <identifier>
+               <system value="http://dktk.dkfz.de/fhir/onco/core/CodeSystem/Projects"/>
+               <value value="http://dktk.dkfz.de/fhir/sid/exliquid-specimen"/>
+            </identifier>
+            <status value="unsatisfactory"/>
```

Comment:
If it is a default value, I would suggest “available”

Reply:
adapted in [d21866547195a34b02ead8de30b9230d5cb455e3](https://github.com/samply/obds2fhir/pull/52/commits/d21866547195a34b02ead8de30b9230d5cb455e3)